libraries/AP_AHRS/AP_AHRS_Logging.cpp

```diff
@@ -53,14 +54,15 @@ void AP_AHRS::Write_Attitude(const Vector3f &targets) const
 {
     const struct log_Attitude pkt{
         LOG_PACKET_HEADER_INIT(LOG_ATTITUDE_MSG),
-        time_us         : AP_HAL::micros64(),
+        time_us         : AP::scheduler().get_loop_start_time_us(),
```

Comment:
This introduces a coupling between AHRS and the scheduler.

We merged a change to use the sample time in the rate message - this uses a different pattern where we don't pass that timestamp in...